app/views/devise/registrations/new.html.erb

```diff
@@ -10,6 +10,14 @@
             <%= f.error_notification %>
 
             <div class="form-inputs">
+              <%= f.input :first_name,
+                          required: true,
+                          autofocus: true,
+                          input_html: { autocomplete: "first_name" }%>
+              <%= f.input :last_name,
+                          required: true,
+                          autofocus: true,
```

Comment:
autofocus only makes sense on 1 form input. Autofocus means this input field will be focused when the page loads.